cli/command/stack/deploy.go

```diff
@@ -63,9 +63,9 @@ func runDeploy(dockerCli command.Cli, opts deployOptions) error {
 	}
 
 	switch {
-	case opts.bundlefile == "" && opts.composefile == "":
+	case opts.bundlefile == "" && len(opts.composefile) == 0:
```

Comment:
i think we should keep the symmetry :)
i mean on checking the emptiness of a string always using len function instead of '== "" '

Reply:
I really dislike checking empty string using `len(string) == 0`. Sure it works, but it's a lot less obvious when reading the code. I think most people say "empty string", not "a string of zero characters", so `variable == ""` matches this phrase.

I'm in favour of keeping it as it is now.